scipy/stats/mstats_basic.py

```diff
@@ -2558,30 +2570,42 @@ def skewtest(a, axis=0):
     y = ma.where(y == 0, 1, y)
     Z = delta*ma.log(y/alpha + ma.sqrt((y/alpha)**2+1))
 
-    return SkewtestResult(Z, 2 * distributions.norm.sf(np.abs(Z)))
+    return SkewtestResult(*scipy.stats.stats._normtest_finish(Z, alternative))
 
 
 KurtosistestResult = namedtuple('KurtosistestResult', ('statistic', 'pvalue'))
 
 
-def kurtosistest(a, axis=0):
+def kurtosistest(a, axis=0, alternative='two-sided'):
     """
     Tests whether a dataset has normal kurtosis
 
     Parameters
     ----------
-    a : array
+    a : array_like
         array of the sample data
     axis : int or None, optional
        Axis along which to compute test. Default is 0. If None,
        compute over the whole array `a`.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+        * 'two-sided': the kurtosis of the distribution underlying the sample
+          is different from that of the normal distribution
+        * 'less': the kurtosis of the distribution underlying the sample
+          is less than that of the normal distribution
+        * 'greater': the kurtosis of the distribution underlying the sample
+          is greater than that of the normal distribution
+
+        .. versionadded:: 1.7.0
 
     Returns
     -------
-    statistic : float
+    statistic : array_like
```

Comment:
More accurately, these would be "scalar or ndarray".

Reply:
> More accurately, these would be "scalar or ndarray". (Could be scalar, and if it's array-like, it's going to be an ndarray)

I think `array_like` includes both scalars and array outputs. So, we can change it that way.

Reply:
I'm not sure if `array_like` is formally defined anywhere, so I guess we can use it to mean whatever we want. But we could be more specific as we push toward better documentation.

Reply:
> I'm not sure if `array_like` is formally defined anywhere, so I guess we can use it to mean whatever we want. But we could be more specific as we push toward better documentation. (Not required for this PR, though.)

I think that NumPy defines `array_like` in https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind . See gh-13621

Reply:
Oh, nice. Thanks for pointing that out. Note that it only specifically says it's for documenting _arguments_, though. We are flexible about arguments, but we know what the return types will be. 

But even e.g. `np.mean` doesn't document this perfectly. It says that the output type will be ndarray
![image](https://user-images.githubusercontent.com/6570539/120694301-e30a3900-c45e-11eb-81cf-dce589fa53d4.png)

but as we know:
```
isinstance(np.mean([[1, 2, 3]], np.ndarray)  # True
isinstance(np.mean([1, 2, 3]), np.ndarray)  # False
```
I'll go ahead and merge this as-is, if it sounds good to you? 
We can consider adding this to the huge list of things we'd like the documentation to be more consistent about.

Reply:
> if it sounds good to you?

Yep, everything sounds good to me!